docker-compose-development.override.yml

```diff
@@ -0,0 +1,15 @@
+version: '2'
```

Comment:
Oh - and just to follow onto getting rid of the .env file, Compose 1.9.0 will have support for default values for environment variables, so we can do something like this once this is released:

```
ports:
    - "${VARNISH_PORT:-80}:80"
```
That lets us have our cake and eat it too by allowing a .env override without requiring it.